forward43/utils/data.py

```diff
@@ -0,0 +1,56 @@
+import os
+import json
+
+
+def get_doc_id(document, source):
+    '''
+    For a given document from source dataset, generate a document id
+    @document : Doument object. There should be a project id associated with each document
+    @source   : Source dataset
+    '''
+    return source + '_' + document['id']
+
+def bulk_json_data(index, data, source):
+    '''
+    Creates bulk actions from a list of json data
+    @index  : index name
+    @data   : list of json data
+    @source : source of the dataset (e.g., kickstarter, ulule, etc.)
+    '''
+    for doc in data:
+
+        _id = get_doc_id(doc, source)
+
+        yield {
+            '_index'  : index,
+            '_id'     : _id,
+            '_source' : doc
+        }
+
+
+def load_data_from_json_file(filename):
```

Comment:
nitpick: `load_data` is overloaded here as we're also loading data into ES as well as to memory: I'd suggest `read_data`